docker-entrypoint.sh

```diff
@@ -43,6 +43,14 @@ if [[ "$1" == apache2* ]] || [ "$1" == php-fpm ]; then
 			EOF
 			chown www-data:www-data .htaccess
 		fi
+	else
+		CONTAINER_WORDPRESS_VERSION=$(php -r "include('/usr/src/wordpress/wp-includes/version.php'); echo \$wp_version;")
+		INSTALLED_WORDPRESS_VERSION=$(php -r "include('wp-includes/version.php'); echo \$wp_version;")
+		if ! [ "$CONTAINER_WORDPRESS_VERSION" == "$INSTALLED_WORDPRESS_VERSION" ]; then
```

Comment:
In regards to putting it behind a flag, wouldn't this issue only occur for users that are pointing to `latest` instead of a set version? Otherwise, the container's version of wordpress would not change and would not trigger an upgrade of files.